packages/core/src/module-resolver.ts

```diff
@@ -1020,6 +1022,19 @@ export class Resolver {
             request,
             request.alias(found?.[0]).rehome(resolve(pkg.root, 'package.json'))
           );
+        } else if (pkg.isEngine()) {
+          let targetingEngine = this.engineConfig(packageName);
+          if (targetingEngine) {
+            let appJSMatch = await this.searchAppTree(
+              request,
+              targetingEngine,
+              request.specifier.replace(packageName, '.')
+            );
+            if (appJSMatch) {
+              console.log('fallbackResolve: non-relative appJsMatch', request.specifier);
```

Comment:
I would probably change this message to say something about engines since it's hyper engine specific 🤔 actually we should remove the console log and update the one below now that I look at it